### DISCUSSION TITLE
heat flux limit and its measurement

### DISCUSSION THREAD

**Discussion by fr-evtech** [firemodels/fds#13775] [Q&A] [answered]
Good morning,
in my country law and fire brigade take as a reference the ISO 13571 for the limit of maximum thermal radiation (from fire, smoke, structures) on people fleeing during a building fire. The limit is 2,5 kW/m2.
Having to use FDS to verify this parameter, I opted for 2 sensor types: gas-phase integrated intensity and gas phase radiative heat flux gas.
By my understanding the integrated intensity one detects all the incoming radiation on a sphere surface (4pi), so even at the start of simulation the default value due to ambient temperature (20°C) is more or less 1,7kW/m2.
Th radiative heat flux gas, instead, detects the radiation on "only" half a sphere (2pi), so it needs to be aligned: this per se can be a problem me, since if I put it vertically then it can easily detect the direct fire radiation, while placing it horizontally allows me to focus on the smoke layer.

The question is: what sensor should I use, and how should I use it?
If I use the integrated intensity, should I subtract the radiation due to ambient temperature (1,7) to the actual measurements before comparing them to the 2,5 limit, since it is often reached after just a few minutes? (even though it sounded wrong while I was writing it...) 
If I use the radiative heat flux gas, how should I align them? Is it somehow possible to sum the heat flux taking into account two or more different sources?
Thanks to whoever helps.

**Comment by drjfloyd** [comment, marked as answer]
I would not use the integrated intensity.  
One option is a sensor pointed at the fire and one at the ceiling and then take the vector sum of the two. (see Chap 66 of the SFPE Handbook).
Another option is multliple sensors pointing along the arc from the fire to the ceiling and picking whichever is the highest at any timestep.

**Reply by fr-evtech** [reply]
Thank you for your answer.
As I thought, the starting radiation is already too high and respect the ISO limit.

Is it correct to assume it's safe to point a sensor vertically only in the proximity of the fire, while I set them horizontally in the remaining parts of the building (i.e along the escape routes)?

**Reply by drjfloyd** [reply]
I suggest you experiment with various approaches and see what gives the results that best meet your specific needs.
